.github/workflows/ci.yml

```diff
@@ -100,6 +100,7 @@ jobs:
       run: |
         cargo update -p ahash --precise 0.8.7
         cargo update -p bumpalo --precise 3.14.0
+        cargo update -p objc2-encode --precise 4.0.3
```

Comment:
Suggestion to see if we can run `cargo +nightly -Zminimal-versions generate-lockfile` before doing an MSRV test? I've been doing that on all my repositories and it at the same time ensures the lower bound of all my dependencies are correct/buildable :)

(Assuming there aren't too many different combinations / features / optional dependencies to test, of course)

Reply:
For my part, feel free to try! I only did this now to fix CI.